<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KappaCalculate</class>
 <widget class="QWidget" name="KappaCalculate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KappaCalculate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Kappa</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="ground_turth_label">
           <property name="text">
            <string>Ground      Truth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_path_truth"/>
         </item>
         <item>
          <widget class="QPushButton" name="InputTurthBtn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Simulation Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_path_compared"/>
         </item>
         <item>
          <widget class="QPushButton" name="InputCompBtn">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QRadioButton" name="radioButton_samplingRate">
               <property name="text">
                <string>Random Sampling</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_samplingAmount">
               <property name="text">
                <string>Unifom Sampling</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Sampling rate for total valid pixels:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_samplingRate"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Numbers of sampling for each land use type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_samplingAmout"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="startBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>51</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>51</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="textAccuracy"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>FoM</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_baseyear_FoM">
           <property name="text">
            <string>Start         Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_path_baseyear"/>
         </item>
         <item>
          <widget class="QPushButton" name="InputBaseyearBtn_FoM">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="ground_turth_label_FoM">
           <property name="text">
            <string>Ground      Truth</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_path_truth_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="InputTurthBtn_FoM">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_endyear_FoM">
           <property name="text">
            <string>Simulation Result</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="input_path_compared_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="InputCompBtn_FoM">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QRadioButton" name="rbtnFoM">
           <property name="text">
            <string>Figure of merit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbtnKappa">
           <property name="text">
            <string>Kappa (Unchanged samples)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QTextEdit" name="textAccuracy_FoM"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnStart_FoM">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>191</height>
            </size>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
